code/modules/power/power.dm

```diff
@@ -289,14 +289,9 @@
 			SEND_SIGNAL(src, COMSIG_MACHINERY_POWER_LOST)
 			. = TRUE
 
-	if(appearance_power_state != (machine_stat & NOPOWER))
+	if(!(machine_stat & NOPOWER))
 		update_appearance()
 
-// Saves like 300ms of init by not duping calls in the above proc
-/obj/machinery/update_appearance(updates)
```

Comment:
what?

Reply:
idk you tell me. What does this comment even mean?

Reply:
What i understand is it kept track of the last state of the machine in its `appearance_power_state`(which checks if the machine is unpowered or not). But we can check for its power state directly in `machine_stat` so this var is redundant with that.

Considering how often `update_appearance()` is called on machines now since we are no longer keeping track of `appearance_power_state` it should be slightly faster.

Nothing is broken from testing in game

Reply:
see review